src/universal/modules/userDashboard/components/Organization/Organization.js

```diff
@@ -83,35 +90,45 @@ const Organization = (props) => {
           />
         </BackControlBlock>
         <AvatarAndName>
-          <PhotoUploadModal picture={pictureOrDefault} toggle={toggle} unstyled>
-            <OrgAvatarInput orgId={orgId} />
-          </PhotoUploadModal>
+          {isBillingLeader ?
+            <PhotoUploadModal picture={pictureOrDefault} toggle={toggle} unstyled>
+              <OrgAvatarInput orgId={orgId} />
```

Comment:
don't let folks change the avatar or the org name unless they are the billing leader